### ISSUE TITLE
[Wotlk] Quest: Gammothra the Tormentor [#11721]

### ISSUE THREAD

**Issue by Argonn** [unified-db/Database#130]
When on this quest, a couple of things are off.
1. The item will not 'shrink' any of the elites. (making this quest un-soloable).
2. Gammothra is standing with his 'behind' towards the entrance.
3. When attacking him (only tested range) he will fly trough the roof. (no kidding).

Quest:
http://www.wowhead.com/quest=11721/gammothra-the-tormentor
Quest item:
http://www.wowhead.com/item=34979
NPC:
http://www.wowhead.com/npc=25789/gammothra-the-tormentor
Weakend NPC:
http://www.wowhead.com/npc=25790/weakened-gammothra

**Comment by Grz3s** [comment]
pls be patient ... I'll look at all Northrend issues... after 408 release*..

**Comment by BeerfishWoW** [comment]
Can we keep on reporting Northrend issues in the meantime, though?

**Comment by Grz3s** [comment]
ofc..

**Comment by Grz3s** [comment]
this one req. spell target fix....
spell: ID - 46010 Bloodspore Ruination
```code
Effect 0: Id 6 (SPELL_EFFECT_APPLY_AURA)
BasePoints = 1
Targets (25, 0) (TARGET_DUELVSPLAYER, NO_TARGET)
Aura Id 12 (SPELL_AURA_MOD_STUN), value = 1, misc = 0 (0), miscB = 0, periodic = 0

Effect 1: Id 6 (SPELL_EFFECT_APPLY_AURA)
BasePoints = 1
Targets (25, 0) (TARGET_DUELVSPLAYER, NO_TARGET)
Aura Id 23 (SPELL_AURA_PERIODIC_TRIGGER_SPELL), value = 1, misc = 0 (0), miscB = 0, periodic = 5000
   Trigger spell (46012) Bloodspore Poison. Chance = 101
   Description: Weakens magnataur.
   ToolTip: Weakened.

Effect 2:  NO EFFECT
```
Somehow ... effect of this spell targets player not ... targeted creature...
We cannot use:
-- DELETE FROM spell_script_target WHERE entry = 46012;
-- DELETE FROM spell_script_target WHERE entry = 46010;
it returns an error...

So spell need to be fixed in core....

Here is all info from sniff.....
http://paste2.org/GGZ70U7a

**Comment by Grz3s** [comment]
sor ..mistake :) ..
Also ... spell must have negative effect (now its positive)

**Comment by xfurry** [comment]
@Grz3s 
Ths spell target works fine.
We need to use eventAI to update the creature entry to 25790 or 25788 on spell hit from aura 46012.

Not sure, if I can test this right now, because @killerwife screwed up the eventAI timer system recently, but I think that my idea should work fine.

**Comment by killerwife** [comment]
https://github.com/cmangos/mangos-tbc/pull/122 was fixed the day after, but not yet merged

**Comment by Grz3s** [comment]
ok... 
I did second test for this core: https://github.com/cmangos/mangos-wotlk/commit/a9f2b4f05212a1b7f93e87b32673f38a18c939b5
And still spells doesn't work like they should.
here how it works:
https://youtu.be/w6X6JPRf5Kw

- we can see that second effect:
```code
Effect 1: Id 6 (SPELL_EFFECT_APPLY_AURA)
BasePoints = 1
Targets (25, 0) (TARGET_DUELVSPLAYER, NO_TARGET)
Aura Id 23 (SPELL_AURA_PERIODIC_TRIGGER_SPELL), value = 1, misc = 0 (0), miscB = 0, periodic = 5000
   Trigger spell (46012) Bloodspore Poison. Chance = 101
   Description: Weakens magnataur.
   ToolTip: Weakened.
```
is applied on player not on creature... (player changes his size,, and aura I applied - this should be done with targeted creature)
Also. 46012 has positive effect on player.. so can be removed by click on it...
On sniff. (and all YT vids) we can see that spell has red frames (negative effect)
```code
[0] Slot: 0
[0] HasAura: True
[0] SpellID: 46012 (46012)
[0] Flags: Negative (16)  -- here
```
https://www.youtube.com/watch?v=5Gk0tAo_gwQ     <-- 1.24 how it works correctly


I don't think that EventAi will fix this issue... (I will use it... but for entry update on spell hit)
